contracts/FTokenStrat.sol

```diff
@@ -159,4 +180,9 @@ contract FTokenStrat is IStrat {
         timelock = Timelock(_newTimelock);
     }
 
+    function changeRewardPool(IRewardPool _newRewardPool ) public onlyStrategist {
```

Comment:
What happens to the funds in the old pool? We need to exit any existing deposits in old pool and move them to the new pool. Also this is a dangerous function that should be assigned to owner instead.